eng/ignore-links.txt

```diff
@@ -8,3 +8,5 @@ https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storage/storage-blob-cha
 https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storage/storage-blob-changefeed/samples/typescript
 https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storage/storage-blob-changefeed/test/
 https://github.com/Azure/azure-digital-twins/blob/private-preview/Documentation/how-to-manage-routes.md
+https://docs.microsoft.com/javascript/api/@azure/arm-webpubsub
```

Comment:
@qiaozha We generally avoid putting things into the ignore-links.txt file. We have guidelines for handling links at http://aka.ms/azsdk/guideline/links. 

Please make sure that you have an issue filed to remove these from this ignore file.

Reply:
Sure. I have created issue #16776